massa-proto/proto/massa/api/v1/execution.proto

```diff
@@ -67,16 +70,226 @@ message ScExecutionEventContext {
 
 // ScExecutionEventStatus type enum
 enum ScExecutionEventStatus {
-  SC_EXECUTION_EVENT_STATUS_UNSPECIFIED = 0; // Defaut enum value
+  SC_EXECUTION_EVENT_STATUS_UNSPECIFIED = 0; // Default enum value
   SC_EXECUTION_EVENT_STATUS_FINAL = 1; // Final status
   SC_EXECUTION_EVENT_STATUS_READ_ONLY = 2; // Read only status
   SC_EXECUTION_EVENT_STATUS_FAILURE = 3; // Failure status
 }
 
 // ExecutionOutputStatus type enum
 enum ExecutionOutputStatus {
-  EXECUTION_OUTPUT_STATUS_UNSPECIFIED = 0; // Defaut enum value
+  EXECUTION_OUTPUT_STATUS_UNSPECIFIED = 0; // Default enum value
   EXECUTION_OUTPUT_STATUS_CANDIDATE = 1; // Candidate status
   EXECUTION_OUTPUT_STATUS_FINAL = 2; // Final status
 }
- 
\ No newline at end of file
+
+// StateChanges
+message StateChanges {
+  // Ledger changes
+  LedgerChanges ledger_changes = 1;
```

Comment:
Why ledger changes and executed ops has a subtype to encapsule the repeated entry and async pool don't have? Is that useful?

Reply:
This a miroir of the existing Rust struct

Yes, we could remove this additional nested struct

Reply:
done